dags/openshift_nightlies/config/install/rosa/ovn.json

```diff
@@ -3,7 +3,8 @@
     "aws_access_key_id": "",
     "aws_secret_access_key": "",
     "rosa_environment": "staging",
-    "rosa_installation_method": "osde2e",
+    "rosa_installation_method": "rosa",
+    "rosa_cli_version": "latest",
```

Comment:
Maybe master instead of latest?

Reply:
Intentionally leaving it at latest - we will only use master if need be,

Reply:
ok, understood

Reply:
@smalleni lets swap the default to master until the new bits are rolled for rosa. Otherwise things will be broken since we merged 103 which looks for that network flag.

Reply:
ack